Sources/Commands/Completions+zsh.swift

```diff
@@ -12,6 +12,9 @@ import Foundation
 import Basic
 import Utility
 
+let dependencies = "swift package show-dependencies | grep -oE '── (.*)<' | sed -E 's/── (.*)</\\1/'"
+let executables = "swift package describe | grep 'Type: executable' -B 3 | grep 'Name:' | sed -e 's/    Name: //'"
```

Comment:
Ah, I just realized this is dynamic and needs to run per package. Maybe the best thing to do is add a dedicated subcommand for autocompletion, so we don't need to run grep and sed. 
Something like: 
`swift package shell-autocomplete --list-executables`
`swift package shell-autocomplete --list-dependencies`